install_config/configuring_openstack.adoc

```diff
@@ -50,6 +50,14 @@ subnet-id = <UUID of the load balancer subnet>
 Consult your OpenStack administrators for values of the `*OS_*` variables, which
 are commonly used in OpenStack configuration.
 
+[IMPORTANT]
+====
+When triggering a rpm-based installation, only the 
```

Comment:
@gpei, if we s/When triggering a rpm-based installation/During installation
and remove the two notes about the containerized installation, will that be correct and address all your concerns?

Reply:
Yeah, and also please address the directory in `Therefore, *_openstack.conf_* must be in *_/etc/origin/_*.` , which should be _/etc/origin/cloudprovider/_

Reply:
@bysnupy, will you please update this note and rebase?

Reply:
@kalexand-rh , I've completed the rebase of the codes.